tools/mpremote/mpremote/main.py

```diff
@@ -333,21 +347,21 @@ def load_user_config():
     config.commands = {}
 
     # Get config file name.
-    path = os.getenv("XDG_CONFIG_HOME")
+    path = None
+    for var in ["XDG_CONFIG_HOME", "HOME", "USERPROFILE"]:
+        path = os.getenv(var)
+        if path:
+            break
     if path is None:
-        path = os.getenv("HOME")
-        if path is None:
-            return config
-        path = os.path.join(path, ".config")
-    path = os.path.join(path, _PROG)
-    config_file = os.path.join(path, "config.py")
+        return config
+    config_file = os.path.join(path, ".config", _PROG, "config.py")
```

Comment:
I'm not sure this logic of finding the path is correct.  Eg prior to this change we had:
1. if XDG_CONFIG_HOME is set then `path="<XDG_CONFIG_HOME>/mpremote"`
2. else if HOME is set then `path="<HOME>/.config/mpremote"`

With this patch, `path` is just `<XDG_CONFIG_HOME>` or `<HOME>`.

Reply:
good catch, Ill rever to an earlier version that keeps the original behavior.
@stinos ,i folded the code based on your earlier review. I assume you agree with Damien ?

Reply:
Right, I misunderstood what it was doing. Could still use the loop which is imo a clearer way of expressing 'try these paths, with this priority' but then the loop items must be tuples like `[("XDG_CONFIG_HOME", "mpremote"), ("HOME", "config/mpremote"), ...]` which is in turn perhaps less clear.

Reply:
- [x] I've updated the code and verified the behavior with a few tests before & after.
        
- [x] Formatting is removed (sorry for that, some tools try to be too clever) 
- [ ] I have a few tests in pytest, happy to add them somewhere , but could not find how / where test for the tools are located.